matRad_sampling.m

```diff
@@ -192,9 +192,13 @@
 %% perform seriel sampling
 % rough estimate of total computation time
 totCompTime = size(pln.multScen.scenForProb,1) * nomScenTime * 1.1;
-fprintf(['Approximate Total calculation time: ', num2str(round(totCompTime / 3600, 2)), ...
-                        'h. Estimated finish: ', datestr(datetime('now') + seconds(totCompTime)), '\n']);
-    
+try
```

Comment:
Why do we need a try catch statement?

Reply:
`datetime` is not implemented in Octave, so I put a try and catch to show estimated finish time when it's on Matlab, and only show Approximate Total Calculation Time when it's on Octave. Travis is not using `matRad_sampling.m` anyway because of build time limit, but I thought I add this for general compatibility with Octave.

Reply:
OK. then better make an if statement that checks if it is octave of matlab!